### PR TITLE
Purchaseable interdictor manudrive crate

### PR DESCRIPTION
**Pull request by Kubius**
[FEAT]
## About the PR 

Adds an interdictor crate for purchase in Cargo. It contains one interdictor component ManuDrive as well as some complementary materials (three cable coils, two steel ingots and a glass block) sufficient for manufacturing one batch of interdictor parts.

## Why's this needed? 

Provides a way to reacquire interdictor schematics in the event somebody cares enough about interdictors to run off with the disk (or the fabricator with the disk gets wrecked).

### CODE REVIEW

File: code/modules/economy/supply_packs.dm [hunk 1]
```diff
@@ -294,6 +294,13 @@ ABSTRACT_TYPE(/datum/supply_packs)
 	containertype = /obj/storage/crate
 	containername = "Station Pressurization Crate"
 
+/datum/supply_packs/interdictor_supply
+	name = "Interdictor Crate"
+	desc = "One (1) officially licensed Spatial Interdictor manufacture data disk. Includes complementary materials (x3 cable coil, x1 steel ingot, x1 glass block)."
+	category = "Engineering Department"
+	cost = 5000
```
Comment (Tarmunora) [inline code/modules/economy/supply_packs.dm:301]:
```suggestion
	cost = 50000
```

**Review by Tarmunora — CHANGES_REQUESTED** [review]
The whole point of manudrives was to limit accessibility of printing some things. Being able to just buy more of them, especially for a trivial price, goes against that philosophy.

**Comment by Kubius** [comment]
Reasonable enough. I don't think I feel strongly enough about this for it to be worthwhile at $50k, and scarcity can be a good thing for sabotage purposes.

**Comment by ThatFiveGuys** [comment]
> The whole point of manudrives was to limit accessibility of printing some things. Being able to just buy more of them, especially for a trivial price, goes against that philosophy.

No it wasnt? While that was one application of manudrives, part of the application was simply to bundle up recipes. Theres a reason why you can copy and paste manudrives with unlimited recipes, because those arent supposed to be as limited.

**Pull request closed without merging**